.github/workflows/nightly_detect_failure.yml

```diff
@@ -254,6 +254,8 @@ jobs:
         echo "Target bump branch: ${BUMP_BRANCH}"
         echo "Using commit SHA: ${SHA}"
         current_version="${NIGHTLY}"
+        # Remove surrounding quotes from BUMP_BRANCH
```

Comment:
Where were these inserted?

Reply:
I think you want to use [this](https://github.com/actions/github-script?tab=readme-ov-file#result-encoding) instead

Reply:
Thanks.